lib/iris/io/__init__.py

```diff
@@ -334,6 +334,12 @@ def save(source, target, saver=None, **kwargs):
         # Save a cube list to netCDF, using the NETCDF4_CLASSIC storage option
         iris.save(my_cube_list, "myfile.nc", netcdf_format="NETCDF3_CLASSIC")
 
+    .. warning:: Saving a cube to a file from which its data was loaded causes
```

Comment:
The warnign text should start on a new line and be separated from the `.. warning::` directive by a blank line. It is common to align the block with the start of the directive name:
```
.. warning::

   The text of the warning goes here...
```